<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepRapMiniHostClass</class>
 <widget class="QMainWindow" name="RepRapMiniHostClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepRap Mini Host</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonXPlus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>131</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>171</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editY">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editZ">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio01">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1mm</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0mm</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>10.0mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonYPlus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonZPlus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonXMinus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonYMinus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonZMinus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHomeX">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHomeY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHomeZ">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHomeAll">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Home all</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBrowse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExecute">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelLeft">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Left: 00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>291</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Extruder temp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>331</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bed temp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTempExtruder">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0°</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTempBed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTempExtruder">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTempBed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn on</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTempExtruder">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>230.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTempBed">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>70.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port/Baud</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editComPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/ttyUSB0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editComBaud">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>19200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEPlus">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEMinus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editF">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonResetHashCounter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Reset Hash Counter</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkExtrudeWhenMoving">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Extrude when moving</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkAutoRefreshTemperatures">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto refresh temperatures</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkEnableHashes">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Hashes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDebugging">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable debug messages</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCom</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonCom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio01</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio1</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio10</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onRadio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonXPlus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonXPlus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonXMinus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonXMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonYMinus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonYMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonYPlus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonYPlus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonZMinus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonZMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonZPlus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonZPlus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEPlus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonEPlus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEMinus</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonEMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHomeX</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonHomeX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHomeY</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonHomeY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHomeZ</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonHomeZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHomeAll</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonHomeAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonGo</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAutoRefreshTemperatures</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onCheckAutoRefreshTemperatures(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkExtrudeWhenMoving</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onCheckExtrudeWhenMoving(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonResetHashCounter</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonResetHashCounter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkEnableHashes</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onCheckEnableHashes(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTempExtruder</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>editTempExtruderChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonTempExtruder</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonTempExtruder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonTempBed</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonTempBed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editTempBed</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>editTempBedChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowse</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonExecute</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonExecute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onButtonStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDebugging</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RepRapMiniHostClass</receiver>
   <slot>onCheckDebugging(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onButtonCom()</slot>
  <slot>onRadio()</slot>
  <slot>onButtonXPlus()</slot>
  <slot>onButtonYPlus()</slot>
  <slot>onButtonZPlus()</slot>
  <slot>onButtonXMinus()</slot>
  <slot>onButtonYMinus()</slot>
  <slot>onButtonZMinus()</slot>
  <slot>onButtonEPlus()</slot>
  <slot>onButtonEMinus()</slot>
  <slot>onButtonGo()</slot>
  <slot>onButtonHomeX()</slot>
  <slot>onButtonHomeY()</slot>
  <slot>onButtonHomeZ()</slot>
  <slot>onButtonHomeAll()</slot>
  <slot>onCheckAutoRefreshTemperatures(int)</slot>
  <slot>onCheckExtrudeWhenMoving(int)</slot>
  <slot>onButtonResetHashCounter()</slot>
  <slot>onCheckEnableHashes(int)</slot>
  <slot>editTempExtruderChanged(QString)</slot>
  <slot>onButtonTempExtruder()</slot>
  <slot>onButtonTempBed()</slot>
  <slot>editTempBedChanged(QString)</slot>
  <slot>onButtonBrowse()</slot>
  <slot>onButtonExecute()</slot>
  <slot>onButtonStop()</slot>
  <slot>onCheckDebugging(int)</slot>
 </slots>
</ui>
